src/libstore/download.cc

```diff
@@ -230,6 +230,11 @@ struct CurlDownloader : public Downloader
                 curl_easy_setopt(req, CURLOPT_SSL_VERIFYHOST, 0);
             }
 
+            Path netrcFile = settings.get("netrc-file",
+               (format("%1%/%2%") % settings.nixConfDir % "netrc").str());
+            curl_easy_setopt(req, CURLOPT_NETRC_FILE, netrcFile.c_str());
```

Comment:
@shlevy nothing bad happens if the file doesn't exists; it continues to work as if the netrc support  was disabled.

About "curl making a copy of its param": Can you clarify a bit what you mean by that?